modules/nw-controlling-dns-pod-placement.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * networking/dns-operator.adoc
+
+[id="nw-controlling-dns-pod-placement_{context}"]
+= Controlling DNS pod placement
+
+The DNS Operator has two daemon sets, one for CoreDNS and one for managing `/etc/hosts`. The daemon set for `etc/hosts` must run on every node host and include an entry for the cluster image registry to support pulling images. Security policies can prohibit communication between pairs of nodes, which prevents the daemon set for CoreDNS from running on every node.
```

Comment:
Maybe: s/sets,/sets:/

?

Reply:
```
s/managing `/etc/hosts`./managing the `/etc/hosts` file./
```

Also in the second sentence, `etc/hosts` is missing the `/` at the front.

Reply:
Ack - updating. Thanks!